app/assets/stylesheets/application.css

```diff
@@ -13,7 +13,50 @@
  *= require_tree .
  *= require_self
  */
-*{
+* {
```

Comment:
you know this is affecting everything, right? Have you checked if this have not caused any undesired side-effect?

Reply:
Yes i have checked, no undesired side effects

Reply:
Yes i have checked, no undesired side effects